src/main/kotlin/com/openlattice/edm/type/EntityTypePropertyMetadata.kt

```diff
@@ -8,6 +8,8 @@ import com.openlattice.client.serialization.SerializationConstants
  * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
  */
 data class EntityTypePropertyMetadata(
-        @JsonProperty(SerializationConstants.TITLE_FIELD) val title: String,
-        @JsonProperty(SerializationConstants.DESCRIPTION_FIELD) val description: String
+        @JsonProperty(SerializationConstants.TITLE_FIELD) var title: String,
```

Comment:
What's the reason behind switching from val to var?

Reply:
I think it's because of the `UpdateEntityTypePropertyMetadataProcessor.java` in conductor-client PR

Reply:
that's it indeed